src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,626 @@
+import _ from 'lodash';
+import React from 'react';
+import fullSchema36 from 'vets-json-schema/dist/28-8832-schema.json';
+
+import dateUI from '../../../common/schemaform/definitions/date';
+import * as address from '../../../common/schemaform/definitions/address';
+import SSNWidget from '../../../common/schemaform/widgets/SSNWidget';
+import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
+import fullNameUI from '../../../common/schemaform/definitions/fullName';
+import ssnUI from '../../../common/schemaform/definitions/ssn';
+import { genderLabels } from '../../../common/utils/labels';
+
+import VerifiedReviewContainer from '../components/VerifiedReviewContainer';
+import {
+  PrimaryAddressViewField,
+  getPage,
+  VAFileNumberDescription
+} from '../helpers';
+
+import initialData from '../../../../../test/disability-benefits/526EZ/schema/initialData';
+
+function isValidZIP(value) {
+  if (value !== null) {
+    return /^\d{9}$/.test(value) || /^\d{5}-\d{4}$/.test(value);
```

Comment:
This needs to be updated as the second half is optional: `^\d{5}(?:(?:[-\s])?\d{4})?$`